docs/development.md

```diff
@@ -0,0 +1,50 @@
+# Develop k8s-dqlite against an active MicroK8s instance
+
+1. Install development tools:
+
+    ```bash
+    sudo snap install go --classic
+    sudo apt update
+    sudo apt install build-essential -y
+    ```
+
+2. Clone k8s-dqlite repository:
+
+    ```bash
+    git clone https://github.com/canonical/k8s-dqlite
+    ```
+
+3. Install MicroK8s on the machine:
+
+    ```bash
+    sudo snap install microk8s --classic
+    ```
+
+4. Wait for MicroK8s to come up:
+
+    ```bash
+    sudo microk8s status --wait-ready
+    ```
```

Comment:
Are we sure we want to recommend Microk8s instead of CK8s here?

Reply:
I initially wanted to do that a few months back. The difference is listed a few lines further down. Now that this is moved into a separate dev doc out of the Readme we could make a second k8s-snap section.